infra/pr_helper.py

```diff
@@ -257,19 +260,25 @@ def get_past_contributors(self, project_path):
 
     return all_contributors
 
-  def is_author_internal_member(self):
-    """Returns if the author is an internal member."""
+  def get_maintainers(self):
+    """Get a list of internal members."""
+    if self._maintainers:
+      return self._maintainers
+
     response = requests.get(f'{BASE_URL}/contents/infra/MAINTAINERS.csv',
                             headers=self._headers)
     if not response.ok:
       return False
```

Comment:
Would it be better to unify the return type of this function? E.g., always returns a set (of maintainers), not a boolean value.

Reply:
oh yeah, true! I didn't notice that, thanks for pointing out.